Client/src/Pages/Incidents/IncidentTable/index.jsx

```diff
@@ -1,218 +1,207 @@
 import PropTypes from "prop-types";
-import {
-	TableContainer,
-	Table,
-	TableHead,
-	TableRow,
-	TableCell,
-	TableBody,
-	Pagination,
-	PaginationItem,
-	Paper,
-	Typography,
-	Box,
-} from "@mui/material";
-
-import ArrowBackRoundedIcon from "@mui/icons-material/ArrowBackRounded";
-import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
-import { useState, useEffect } from "react";
-import { useSelector } from "react-redux";
-import { networkService } from "../../../main";
```

Comment:
I mean, this whole component seems to have been deleted, maybe there was a problem when merging?

Reply:
You are right! It happens in the @ajhollid merge. I think.

If you go to my first commit. You will see the changes correctly.  https://github.com/bluewave-labs/bluewave-uptime/pull/972/commits/15c4d2b40058372b266f8a1220e3b2144afea155

I will fix it. Thanks.

Reply:
I'm not exactly understanding what's happening, but every time after a merge, my code seems to stop working.

Now I'm getting a 500 error from the backend: 
`{"level":"error","message":"checksQuery is not defined","method":"getTeamChecks","service":"checkController","timestamp":"2024-10-20T15:42:48.177Z"}`

That's strange; I haven't changed anything related to the request, and there were no conflicts with other files during the merge.

Reply:
> You're right! The issue seems to occur in the merge by @ajhollid.
> 
> If you check my first commit (15c4d2b), you'll see the changes reflected correctly.
> 
> I'll fix this. Thanks!

I don't recall merging any code that removes a component from, could you link to the diff so I can check it out?

Reply:
> I'm not exactly understanding what's happening, but every time after a merge, my code seems to stop working.
> 
> Now I'm getting a 500 error from the backend: 
> `{"level":"error","message":"checksQuery is not defined","method":"getTeamChecks","service":"checkController","timestamp":"2024-10-20T15:42:48.177Z"}`
> 
> That's strange; I haven't changed anything related to the request, and there were no conflicts with other files during the merge.

Were you able to figure out what was going on here? If not I'll look into it today

Reply:
> I'm not exactly understanding what's happening, but every time after a merge, my code seems to stop working.
> 
> Now I'm getting a 500 error from the backend: `{"level":"error","message":"checksQuery is not defined","method":"getTeamChecks","service":"checkController","timestamp":"2024-10-20T15:42:48.177Z"}`
> 
> That's strange; I haven't changed anything related to the request, and there were no conflicts with other files during the merge.

@rog404 fixed the issue, there was a missing variable declaration in one of the modules.  If you merge develop in to you r branch you should be good to go.

Reply:
Thanks!